spec/segmenter.html

```diff
@@ -408,6 +443,7 @@ <h1>FindBoundary ( _segmenter_, _string_, _startIndex_, _direction_ )</h1>
         1. If a boundary is found, return the count of code units in _string_ preceding it.
         1. Return _len_.
       </emu-alg>
+      <emu-note>Boundary determination is implementation-dependent, but general default algorithms are specified in <a href="https://unicode.org/reports/tr29/">Unicode Standard Annex #29</a>. It is recommended that implementations use locale-sensitive tailorings such as those provided by the Common Locale Data Repository (available at <a href="https://cldr.unicode.org">https://cldr.unicode.org</a>).</emu-note>
```

Comment:
I've moved this note, because otherwise it gets inserted between the parameter description text and the auto-generated "It performs the following steps when called:" text.